Ownership/TC - LTC/London TC LOREL.txt

```diff
@@ -64,6 +64,7 @@ GUEST:TCNE:*:*
 GUEST:TCSE:EGKK:*
 GUEST:TCS:EGKK:*
 GUEST:TC:EGKK:*
+GUEST:LSF:EGKK:*
```

Comment:
Seaford will never own TC SE, but we do need to reflect the change in hands of VATON!

```suggestion
GUEST:LDL:EGKK:*
GUEST:LD:EGKK:*
```